src/jit/lsra.h

```diff
@@ -584,7 +584,7 @@ class LinearScan : public LinearScanInterface
     bool            isRegCandidate(LclVarDsc* varDsc);
 
     // Return the registers killed by the given tree node.
-    static regMaskTP getKillSetForNode(Compiler* compiler, GenTree* tree);
```

Comment:
This needs to remain static: it's normally used in some debug checks in codegenxarch that I `#if 0`'d out for the time being.